linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -296,45 +304,66 @@ impl LocalKubernetesNet {
             base_dir.join("genesis.json"),
         )?;
 
-        for i in 0..self.num_initial_validators {
-            let cluster_id = self.kind_clusters[i].id();
-            self.kind_clusters[i]
-                .load_docker_image(docker_image.get_name())
-                .await?;
+        let mut kubectl_instance = self.kubectl_instance.clone();
+        let mut tmp_dir_path = self.tmp_dir.path().to_path_buf();
+        let num_shards = self.num_shards;
 
-            let server_config_filename = format!("server_{}.json", i);
-            fs::copy(
-                self.tmp_dir.path().join(&server_config_filename),
-                base_dir.join(&server_config_filename),
-            )?;
-
-            HelmRelease::install(
-                String::from("linera-core"),
-                &base_dir,
-                i,
-                &github_root,
-                self.num_shards,
-                cluster_id,
-            )
-            .await?;
+        let mut validators_initialization_futures = Vec::new();
+        for (i, kind_cluster) in self.kind_clusters.iter().cloned().enumerate() {
+            let docker_image_name = docker_image.get_name().to_string();
+            let base_dir = base_dir.clone();
+            let github_root = github_root.clone();
+
+            let next_kubectl_instance = kubectl_instance.clone();
+            let next_tmp_dir_path = tmp_dir_path.clone();
```

Comment:
Not sure if this is the best way to do this (doing the clones like this and all that), but please feel free to educate me :)

Reply:
This is not totally surprising. One has to provide the `async move` with values to move out.